code/modules/shuttle/navigation_computer.dm

```diff
@@ -126,16 +136,20 @@
 
 /obj/machinery/computer/camera_advanced/shuttle_docker/give_eye_control(mob/user)
 	. = ..()
-	if(QDELETED(user) || !user.client)
+	if(!.)
 		return
+
 	var/mob/camera/aiEye/remote/shuttle_docker/the_eye = eyeobj
 	var/list/to_add = list()
 	to_add += the_eye.placement_images
 	to_add += the_eye.placed_images
 	if(!see_hidden)
 		to_add += SSshuttle.hidden_shuttle_turf_images
 	user.client.images += to_add
+	//Parent proc prevents any changes to view size, so got to toggle it; not using the toggle procs because they also do apply()
+	user.client.view_size.supress_changes = FALSE
 	user.client.view_size.set_view_radius_to(view_range)
+	user.client.view_size.supress_changes = TRUE
```

Comment:
????

Reply:
I don't remember the exact details of `apply()` but as the comment says, it prevented changing the view size if used.